app/src/main/java/app/revanced/integrations/youtube/swipecontrols/controller/gesture/core/VolumeAndBrightnessScroller.kt

```diff
@@ -77,10 +79,22 @@ class VolumeAndBrightnessScrollerImpl(
             ),
         ) { _, _, direction ->
             screenController?.run {
-                if (screenBrightness > 0 || direction > 0) {
-                    screenBrightness += direction
+                if (!Settings.SWIPE_ENABLE_AUTO_BRIGHTNESS.get()){
```

Comment:
This needs refactoring. Settings shouldn't be called here. Theres a dedicated class where they should be saved to variables.